src/v2/guide/components-props.md

```diff
@@ -1,39 +1,39 @@
 ---
-title: Props
+title: Propiedades
 type: guide
 order: 102
 ---
 
-> This page assumes you've already read the [Components Basics](components.html). Read that first if you are new to components.
+> Esta página asume que usted ya ha leído [Conceptos Básicos de Componentes](components.html). Lea eso primero si usted es nuevo con respecto a componentes.
 
-## Prop Casing (camelCase vs kebab-case)
+## Formateando las "Props" (camelCase vs kebab-case)
 
-HTML attribute names are case-insensitive, so browsers will interpret any uppercase characters as lowercase. That means when you're using in-DOM templates, camelCased prop names need to use their kebab-cased (hyphen-delimited) equivalents:
+Los nombres de atributos HTML no distinguen entre mayúsculas y minúsculas, por lo que los navegadores interpretarán los caracteres en mayúscula como en minúscula. Eso significa que cuando usa plantillas del DOM, los nombres de propiedades en formato camelCase necesitan usar sus equivalentes en formato kebab (delimitados por guiones):
 
 ``` js
 Vue.component('blog-post', {
-  // camelCase in JavaScript
+  // formato camelCase en JavaScript
   props: ['postTitle'],
   template: '<h3>{{ postTitle }}</h3>'
 })
 ```
 
 ``` html
-<!-- kebab-case in HTML -->
-<blog-post post-title="hello!"></blog-post>
+<!-- forato kebab en HTML -->
+<blog-post post-title="hola!"></blog-post>
 ```
 
-Again, if you're using string templates, this limitation does not apply.
+Nuevamente, si está utilizando plantillas de cadenas de texto, esta limitación no se aplica.
 
-## Prop Types
+## Tipos de propiedades
 
-So far, we've only seen props listed as an array of strings:
+Hasta ahora, solo hemos visto los accesorios listados como un arreglo de cadenas de texto:
```

Comment:
If you agree, change `los accesorios listados` to `propiedades listadas`

Reply:
Thanks @sxcamacho . Great catch! I did the change. To make your life easier this time, simply pushed a new commit with the change.

Have a great weekend!